pkg/kudoctl/packages/types.go

```diff
@@ -67,6 +88,15 @@ func (p Parameters) Less(x, y int) bool {
 	return p[x].Name < p[y].Name
 }
 
+type Groups []Group
+
+type Group struct {
+	Name        string `json:"name,omitempty"`
+	DisplayName string `json:"displayName,omitempty"`
```

Comment:
why do we need displayName? 🤔

Reply:
To allow a nice human readable title for the group, i.e.
```
Name: "readiness"
DisplayName: "Readiness Parameters"
```